{{THIS-China/CSS}}{{THIS-China/CSS2}}{{THIS-China/Header}}
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8" />
<title>Lab Notebook</title>


  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Lab Notebook</title>

    <link rel="stylesheet" href="styles.css" />
  </head>
  <body>
    <div id="mywrapper">
      
      <div id="top-pic-wrapper"><img src="https://2021.igem.org/wiki/images/f/f1/T--THIS-China--MAINLOGO.jpg" alt="fork me"></div>
      <div id="content-wrapper">
        <div id="mycontent-noside">
          <h3>Weeks 1&2 (May 29 – June 12)</h3>
            <div class="text">
              <font size="+1">1. Preparation of experiment, getting familiar with the lab techniques and lab apparatus. <br>2. Start construction of circuit, preparing individual parts of circuit.
</p>
            </div>
<br>
            <h3>Week 3 (June 13 – 19)</h3>
            <div class="text">
              <font size="+1">1. Construction of the circuit <br> &nbsp; &nbsp; &nbsp; - comE, comD, and cotA in vector (PDG-148) <br>Important issues: results from the first experiment session of the third week show that the construction of our genetic circuit has not been successful. We therefore changed our strategy to first combining the comE and comD with the vector. <br>2. Construct circuit #1 <br> &nbsp; &nbsp; &nbsp; - comE and comD in vector (PDG-148)
</p>
            </div><br>
            <h3>Week 4 (June 20 – 26)</h3>
            <div class="text">
              <font size="+1">1.Construction of circuit #1 using Gibson assembly <br>
Important issues: The results from gel electrophoresis show the correct base pair numbers. But the sequencing test obtained show different results from the expected design of our circuit. We therefore decided to switch and use the traditional restriction enzyme site cutting protocol. <br>
&nbsp;&nbsp;&nbsp;- Redesigned the primers for our new protocol. <br>
2. Construct circuit #1 <br>
&nbsp;&nbsp;&nbsp;- Using restriction enzymes instead of Gibson assembly for the genes comE and comD and the plasmid PDG-148.
</p>
            </div><br>
<h3>Week 5 (June 27 – July 3)</h3>
            <div class="text">
              <font size="+1">1. Successful construction of circuit #1<br>
2. Constructed circuit #2 <br>
&nbsp; &nbsp; &nbsp;- Restriction enzymes sites BamHI-HF and Xhol were used to cut and connect cotA in plasmid. 
3. Verification of sequencing through TIAN prep Mini Plasmid <br>Kit and Nanovue. <br>
4. Gel electrophoresis show successful construction of circuit #2.
</p>
            </div>
<br>
<h3>Week 6 (July 4 – 11)</h3>
            <div class="text">
              <font size="+1">1. Thought of substituting cotA protein with other direct color making chromoproteins <br>
2. PCR amplification of different color chromoproteins, including pink, purple, green, yellow, and blue. <br>
3. Transform them to plasmids with comE and comD into E.coli and conduct transformation to Bacillus subtilis 168. <br>
Important issues: Bacillus subtilis 168 is working badly because they are extremely hard to cultivate. Therefore, according to research from the group STPZ-shenzhen, we decide to use E.coli. 
</p>
            </div><br>
<h3>Week 7 (July 12-17)</h3>
            <div class="text">
              <font size="+1">1. The competent cell DH5alpha demonstrated an effective completion of the circuit, and the trial with purple chromoprotein showed a faint color. To aim for better expression of the colors, we tried DH5a with different concentration levels, hopefully observing a pattern. <br>
2. Transformation and growth of multiple competent cells including Rosetta, DH5a, and Bu21-pog148. Observe expressed colors. Results show that only purple had a faint color. <br>

</p>
            </div><br>
<h3>Week 8 (July 18 – 24)</h3>
            <div class="text">
              <font size="+1">1. Insert plasmids with yellow, green, and pink chromoprotein genes again into bacillus. Attempt for expression of fluorescent gene and observe effectiveness. Tested them with negative control. <br>
Important issue: First, the other colored chromoprotein is not successfully expressing (the color is not dark enough for it to be easily seen). Second, the promotor nlmc is leaking when these chromoproteins are expressed because they are too strong. We decide not to use chromoprotein anymore for substitution. 

</p>
            </div><br>
<h3>Week 9 (July 25 – 31)</h3>
            <div class="text">
              <font size="+1">1. Transformation and growth of bacteria with circuits #1 and #2 together (putting comE, comD, and cotA together). <br>
2. Gel electrophoresis passed, but the protein electrophoresis for comD and comE did not appear, which means that these two proteins did not express. <br>
Important issue: to solve the problem of protein not expressing successfully, we tried to use two plasmids with constructed circuit to increase the chances of expression for comE and comD. One plasmid with comE and cotA gene and the other with comD separately. 

</p>
            </div><br>
<h3>Week 10 (August 1 – 7) </h3>
            <div class="text">
              <font size="+1">1. Growth of cell with pET-21 and pET-28 vectors (substituting the pdg-148) with different gene and resistance. <br>
2. Vector pET-21 and gene comE and cotA is combined then transformed to the ecoli-BL21. <br>
3. The transformation is successful and passes the gel.

</p>
            </div><br>

<h3>Week 11 (August 8 – 14) </h3>
            <div class="text">
              <font size="+1">1. Vector pET-28 and gene comD is combined and PCR of plasmid then transformed into competent cell ecoli-BL21. <br>
2. The transformation is successful and passes the gel.
</p>
            </div><br>

<h3>Week 12 (August 15 – 21)</h3>
            <div class="text">
              <font size="+1">1. Transformation of the 2 plasmids together into one E.coli. <br>
2. Went through protein gel electrophoresis and found out that the protein expressed properly.

</p>
            </div><br>
<h3>Week 13 (August 22 – 28)</h3>
            <div class="text">
              <font size="+1">1. Do different groups to check the whole circuit by adding RR color for it to degrade. <br>
&nbsp; &nbsp; &nbsp;- Five groups, each with: only cotA, cotA and IPTG, comDE and cotA, comDE and cotA and IPTG, and comDE and cotA and IPTG and CSP-1. <br>
2. Found out that the cotA degradation protein works separately. 
Important issue: CSP cannot be detected by the comDE system because they do not express. Solution is proposed in the Parts and Future Goals.

</p>
            </div><br>

<h3>Week 14 (August 29 – September 5)</h3>
            <div class="text">
              <font size="+1">1.Test the penetrability of CSP-1 in 5KD filters in the presence of bacteria with sizes 0.8μm~500μm. 

</p>
            </div><br>

<h3>Week 15 (September 26– October 1)</h3>
            <div class="text">
              <font size="+1">1. Conduct experiments to check whether the comD protein is correctly on the membrane or not.

</p>
            </div><br>


        </div>
      </div>
    </div>
  </body>

</p>

</div>		            <div class="visualClear"></div>
	            </div>
		    </div>

        </div>
    </div>
</html>
